apps/service/src/pages/worldcup.tsx

```diff
@@ -1,66 +1,109 @@
+    React.useState<DrinkWithRound | null>(null);
   const [isBottomSheetOpened, setBottomSheetOpened] = React.useState(false);
+  const [worldCupState, setWorldCupState] = useRecoilState(recoilWorldCupState);
+  const navigate = useNavigate();
+  const {
+    updateToNextRoundState,
+    revertToPrevRoundState,
+    getCurrentCandidate,
+    getTitle,
+  } = useWorldCup();
+  const candidateDrinks = getCurrentCandidate();
 
-  const handleDrinkCardClick = (drink: Drink) => {
+  const handleDrinkCardClick = (drink: DrinkWithRound) => {
     if (drink === selectedDrink) setSelectedDrink(null);
     else setSelectedDrink(drink);
   };
 
+  const handleClickBottomButton = () => {
+    const { currentRound } = worldCupState;
+
+    updateToNextRoundState(selectedDrink?.id!);
+    setSelectedDrink(null);
+    if (isWinnerSelectRound(currentRound)) {
+      // 우승자인 경우, 결과 페이지로 이동한다.
+      navigate.push("/result");
+    }
+  };
+
+  const handleClickHeaderPrevButton = () => {
+    revertToPrevRoundState();
+  };
+
+  React.useEffect(() => {
+    // FIXME: React-Query를 연동 후 해당 부분 변경해야 함
+    // 2인 경우는 월드컵 결과 페이지에서 뒤돌아갔을 때 임시 방편으로 업데이트 되는 것 막아놓기
+    if (worldCupState.currentRound !== 2) {
+      setWorldCupState((prev) => ({
+        ...prev,
+        drinks: DRINK_CARDS.map((drink) => ({
+          ...drink,
+          rounds: [prev.totalRound],
+        })),
+      }));
+    }
+  }, []);
+
   return (
-      <>
-        {isBottomSheetOpened && selectedDrink && (
-            <DrinkInfoBottomSheet
-                drinkCardIcon={"typeA"}
-                drinkName={selectedDrink.name}
-                drinkInformation={selectedDrink.info}
-                drinkMetaData={
-                  {abv: selectedDrink.abv, categoryName: selectedDrink.category.name, origin: selectedDrink.origin}
-                }
-                onClose={() => setBottomSheetOpened(false)}
-            />
-        )}
-        <Layout>
-          <Header type="prev" title="8강"/>
-          <TitleWrapper>
-            <Text type="h1" textAlign="center">
-              비 오는 날씨엔
-            </Text>
-            <Text type="h1" textAlign="center">
-              어떤 술이 더 끌리나요?
-            </Text>
-          </TitleWrapper>
-          <CandidateWrapper>
+    <>
+      {isBottomSheetOpened && selectedDrink && (
+        <DrinkInfoBottomSheet
+          drinkCardIcon={"typeA"}
+          drinkName={selectedDrink.name}
+          drinkInformation={selectedDrink.info}
+          drinkMetaData={{
+            abv: selectedDrink.abv,
+            categoryName: selectedDrink.category,
+            origin: selectedDrink.origin,
+          }}
+          onClose={() => setBottomSheetOpened(false)}
+        />
+      )}
+      <Layout>
+        <Header
+          type="prev"
+          title={getTitle()}
+          onClickIcon={handleClickHeaderPrevButton}
+        />
+        <TitleWrapper
+          title={"비 오는 날씨엔 어떤 술이 더 끌리나요?"}
+          textAlign="center"
+        ></TitleWrapper>
```

Comment:
```suggestion
        <TitleWrapper
          title={"비 오는 날씨엔 어떤 술이 더 끌리나요?"}
          textAlign="center"
        />
```
편하게 써제션 달아드릴게요 ㅎ
`<TitleWrapper />` 랑 `<Title />` 사용하는 부분 통일성 있게 가져가면 좋을 것 같은데 제가 작업할때 같이 반영해보겠습니당~!

Reply:
suggestion은 안 써봤는데 좋은 기능이군요

Reply:
@huewilliams 신우 겁나 많이 쓰게될듯 ㅋ ㅋ 써제션러

Reply:
너무좋다.....감사합니다!